factor/totp/lang/en/factor_totp.php

```diff
@@ -38,6 +42,8 @@
 $string['privacy:metadata'] = 'The TOTP Factor plugin does not store any personal data';
 $string['settings:secretlength'] = 'TOTP secret key length';
 $string['settings:secretlength_help'] = 'Generated TOTP secret key string length';
+$string['settings:window'] = 'TOTP verification window';
+$string['settings:window_help'] = 'The window of time codes are allowed to be verified, forwards and backwards.';
```

Comment:
admin_setting_configduration allows anything, so I think just add some words here that the window will be rounded to the nearest 30 seconds

Reply:
How long each code is valid for. You can set this to a higher value as a workaround if your users device clocks are often slightly wrong. Rounded down to nearest 30 seconds which is the time between new generated codes.